crd/fleet/fleet-crds.yaml

```diff
@@ -4072,11 +5694,37 @@ spec:
                 description: 'Internal: Whether the resource was reconciled by the
                   controller.'
                 type: boolean
+              startTime:
+                description: StartTime is the time that the failover operation started.
+                format: date-time
+                type: string
+              state:
+                description: State is the current state of the failover operation.
+                  The values are `InProgress`, `Success`, `Failed_RollbackInProgress`,
+                  `Failed_RollbackSuccess`, `Failed_RollbackFailed` `InProgress` means
+                  the failover is still in progress. `Success` means that the failover
+                  has completed. It is complete when the new primary instance is successfully
+                  promoted. `Failed_RollbackInProgress` means that the operator was
+                  unable to promote the new primary instance, and is attempting to
+                  restart the old primary instance. `Failed_RollbackSuccess` means
+                  that the operator was unable to promote the new primary instance,
+                  and successfully restarted the old primary instance. `Failed_RollbackFailed`
+                  means that  the operator was unable to promote the new primary instance,
+                  and were not able to restart the old primary instance. The DBCluster
```

Comment:
Replace "were not able to restart" with "and that the operator wasn't able to restart..."